style.json

```diff
@@ -4055,23 +4055,34 @@
           [
             "in",
             "subclass",
+            "artwork",
             "bollard",
+            "bicycle_rental",
             "border_control",
             "cycle_barrier",
+            "ferry_terminal",
             "gate",
             "lift_gate",
+            "playground",
+            "pitch",
             "post_box",
+            "post_office",
             "recycling",
             "sally_port",
             "shelter",
             "stile",
+            "taxi",
             "telephone",
             "toll_booth",
+            "toilets",
+            "subway_entrance",
+            "train_station_entrance",
             "waste_basket"
           ],
           [
             "in",
             "class",
+            "information",
```

Comment:
this will need to be improved in the future: it also includes the information offices (but for now, they are not imported by Fafnir, so it's ok)